File: Task_1/code/java/MatrixUtils.java
/**
 * Matrix Utilities for Java
 */
import java.util.Random;

public class MatrixUtils {
    
    /**
     * Create a random matrix of given size
     */
    public static double[][] createRandomMatrix(int size) {
        Random random = new Random();
        double[][] matrix = new double[size][size];
        
        for (int i = 0; i < size; i++) {
            for (int j = 0; j < size; j++) {
                matrix[i][j] = random.nextDouble();
            }
        }
        
        return matrix;
    }
    
    /**
     * Create a zero matrix of given size
     */
    public static double[][] createZeroMatrix(int size) {
        return new double[size][size];
    }
}

File: Task_1/code/python/matrix_utils.py
"""
Matrix Utilities for Python
"""
import random


def create_random_matrix(size):
    """Create a random matrix of given size"""
    return [[random.random() for _ in range(size)] for _ in range(size)]


def create_zero_matrix(size):
    """Create a zero matrix of given size"""
    return [[0.0 for _ in range(size)] for _ in range(size)]